src/main/java/com/bank/spring/jpa/saldo/SaldoService.java

```diff
@@ -0,0 +1,29 @@
+package com.bank.spring.jpa.saldo;
+
+import com.bank.spring.jpa.client.model.Client;
+import com.bank.spring.jpa.saldo.model.Saldo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SaldoService {
+
+    @Autowired
+    private SaldoRepository saldoRepository;
+
+    public SaldoService(SaldoRepository saldoRepository) {
+        this.saldoRepository = saldoRepository;
+    }
+
+    public void generateSaldo(Client client) {
+        Saldo saldo = new Saldo(client,"PLN", 0);
+    }
```

Comment:
Nie uruchamiałem tego, ale.
Czyli jak uzytkownik dokona transakcji to teraz pierw wygeneruje sie puste saldo z hardcodowanymi danymi.
A potem dopiero zmienimy na wlasciwe dane?
Trzeba bedzie rozkminić jak zrobić to inaczej, zeby za jednym strzałem dodać odrazu poprawną walute.
Zaraz zabieram się za analize.